services/blockchain-connector/methods/delegates.js

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Can we please rename the file to `services/blockchain-connector/methods/dpos.js` so that all the module related methods are put together?